libdevcore/Exceptions.h

```diff
@@ -32,6 +32,9 @@
 #include "CommonData.h"
 #include "FixedHash.h"
 
+#pragma warning( push )
+#pragma warning(disable:811) //exception specification for implicitly declared virtual function
```

Comment:
What are the implicitly declared virtual functions?    We should just declare them properly, with the same exception specs are the parent class.